qiling/os/windows/dlls/ntdll.py

```diff
@@ -103,7 +103,88 @@ def hook_NtQueryInformationProcess(ql, address, params):
     # TODO have no idea if is cdecl or stdcall
 
     _QueryInformationProcess(ql, address, params)
+def _QuerySystemInformation(ql, address, params):
+    siClass = params["SystemInformationClass"]
+    pt_res = params["ReturnLength"]
+    dst = params["SystemInformation"]
+    if (siClass == SystemBasicInformation):
+        bufferLength = params["SystemInformationLength"]
+        if (ql.archtype == QL_ARCH.X8664):
+            sbi = qiling.os.windows.structs.SystemBasicInforation(ql,
+                                                              Reserved=0,
+                                                              TimerResolution = 156250 ,
+                                                              PageSize=ql.os.heap.page_size,
+                                                              NumberOfPhysicalPages = 0x003FC38A,
+                                                              LowestPhysicalPageNumber=1,
+                                                              HighestPhysicalPageNumber=0x0046DFFF,
+                                                              AllocationGranularity=1,
+                                                              MinimumUserModeAddress=0x10000,
+                                                              MaximumUserModeAddress=0x7FFFFFFEFFFF,
+                                                              ActiveProcessorsAffinityMask = 0x3F,
+                                                              NumberOfProcessors = 0x6)
+        elif(ql.archtype == QL_ARCH.X86):
```

Comment:
please leave a space after `elif`

also on this line, you can remove `(` and `)`

Reply:
thanks a lot for fix my coding style, do you have any reference about python coding standard. I'm a newbie in software development.